docs/JestObjectAPI.md

```diff
@@ -428,9 +428,9 @@ When this API is called, all pending micro-tasks that have been queued via `proc
 
 ### `jest.runAllTimers()`
 
-Exhausts the **macro**-task queue (i.e., all tasks queued by `setTimeout()`, `setInterval()`, and `setImmediate()`).
+Exhausts both the **macro**-task queue (i.e., all tasks queued by `setTimeout()`, `setInterval()`, and `setImmediate()`) and the **micro**-task queue (usually interfaced in node via `process.nextTick`).
```

Comment:
While we're at enumerating examples, let's add Promises next to `process.nextTick` as they're used way more often and

Reply:
By default, we don't exhaust it though - only if you've replaced `Promise` with a fake one

Reply:
Oh, that's a good point 🙈 we should at some point tho

Reply:
#6876